packages/react-components/react-accordion/src/stories/AccordionCollapsible.stories.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import { AccordionItem, AccordionHeader, AccordionPanel, Accordion, AccordionProps } from '../index';
+import { AccordionItem, AccordionHeader, AccordionPanel, Accordion } from '../index';
 
-export const Collapsible = (args: AccordionProps) => (
-  <Accordion {...args}>
+export const Collapsible = () => (
+  <Accordion collapsible>
```

Comment:
The same as #23231. Our export to CodeSandbox does not support `.args` => all examples look the same as `args` is `undefined`.